src/document.h

```diff
@@ -205,7 +205,9 @@ struct Document {
                 if (image.trefc) {
                     fos.PutC(image.image_type);
                     sos.WriteDouble(image.display_scale);
-                    fos.Write(image.image_data.data(), image.image_data.size());
+                    wxInt32 imagelen(image.image_data.size());

```

Comment:
We don't expect images >2GB.. yet, to be correct, the result of `size()` (a 64-bit number) would need to be checked, to be correct.

Instead, since images are large, why don't we just store this length as 64-bit? No need for error checking, and future proof should anyone ever want to store larger images.

Reply:
That might be a very huge image but you are absolutely right! Thanks :-)